qiskit_experiments/library/randomized_benchmarking/layer_fidelity.py

```diff
@@ -76,6 +76,50 @@ class LayerFidelity(BaseExperiment, RestlessMixin):
 
     # section: reference
         .. ref_arxiv:: 1 2311.05933
+
+    # section: example
+        .. jupyter-execute::
+            :hide-code:
+
+            # backend
+            from qiskit_aer import AerSimulator
+            from qiskit_ibm_runtime.fake_provider import FakePerth
+            backend = AerSimulator.from_backend(FakePerth())
+
+        .. jupyter-execute::
+
+            import numpy as np
+            from qiskit_experiments.library import StandardRB
+            from qiskit_experiments.library.randomized_benchmarking import LayerFidelity
+
+            lengths = np.arange(1, 800, 200)
+            two_qubit_layers=[[(0, 1), (3, 5)], [(1, 3), (5, 6)]]
+
+            for qpair in two_qubit_layers:
+                for qpair_ in backend.coupling_map:
+                    if qpair_ == qpair:
+                        pass
+
+            num_samples = 6
+            seed = 1010
+
+            exp = LayerFidelity(
+                    physical_qubits=[0, 1, 3, 5, 6],
+                    two_qubit_layers=two_qubit_layers,
+                    lengths=lengths,
+                    backend=backend,
+                    num_samples=num_samples,
+                    seed=seed,
+                    two_qubit_gate=None,
+                    one_qubit_basis_gates=None,
+            )
+
+            exp.set_run_options(shots=5000)
+            expdata = exp.run(backend_run=True).block_for_results()
+            results = expdata.analysis_results()
+
+            for i in range(4):
```

Comment:
Note that there are 6 figures because there are a couple of single qubit RB seqeuences run on the idle edge qubits while the other pairs are run.

Also, maybe you want to print out the the `EPLG` result?

Reply:
done.